apps/mobile/src/utils/localization/currency.ts

```diff
@@ -0,0 +1,80 @@
+    HKD: {code: 'HKD', flag: '🇭🇰', name: '港幣', symbol: 'HK$', position: 'before'},
+    HUF: {code: 'HUF', flag: '🇭🇺', name: 'Magyar forint', symbol: 'Ft', position: 'after'},
+    IDR: {code: 'IDR', flag: '🇮🇩', name: 'Rupiah', symbol: 'Rp', position: 'before'},
+    ILS: {code: 'ILS', flag: '🇮🇱', name: 'שקל חדש', symbol: '₪', position: 'before'},
+    INR: {code: 'INR', flag: '🇮🇳', name: 'Indian rupee', symbol: '₹', position: 'before'},
+    JPY: {code: 'JPY', flag: '🇯🇵', name: '日本円', symbol: '¥', position: 'before'},
+    KRW: {code: 'KRW', flag: '🇰🇷', name: '대한민국 원', symbol: '₩', position: 'before'},
+    MXN: {code: 'MXN', flag: '🇲🇽', name: 'Peso mexicano', symbol: 'Mex$', position: 'before'},
+    NOK: {code: 'NOK', flag: '🇳🇴', name: 'Norske kroner', symbol: 'kr', position: 'before'},
+    NZD: {code: 'NZD', flag: '🇳🇿', name: 'New Zealand dollar', symbol: 'NZ$', position: 'before'},
+    PLN: {code: 'PLN', flag: '🇵🇱', name: 'Złoty', symbol: 'zł',position: 'after'},
+    RON: {code: 'RON', flag: '🇷🇴', name: 'Leu românesc', symbol: 'lei', position: 'after'},
+    RSD: {code: 'RSD', flag: '🇷🇸', name: 'Српски динар', symbol: 'дин.', position: 'after'},
+    RUB: {code: 'RUB', flag: '🇷🇺', name: 'Российский рубль', symbol: '₽', position: 'after'},
+    SAR: {code: 'SAR', flag: '🇸🇦', name: 'ريال سعودي', symbol: 'ر.س', position: 'before'},
+    SEK: {code: 'SEK', flag: '🇸🇪', name: 'Svensk krona', symbol: 'kr', position: 'before'},
+    SGD: {code: 'SGD', flag: '🇸🇬', name: 'Singapore dollar', symbol: 'S$', position: 'before'},
+    THB: {code: 'THB', flag: '🇹🇭', name: 'Thai baht', symbol: '฿', position: 'before'},
+    TRY: {code: 'TRY', flag: '🇹🇷', name: 'Türk lirası', symbol: '₺', position: 'before'},
+    UAH: {code: 'UAH', flag: '🇺🇦', name: 'Українська гривня', symbol: '₴', position: 'after'},
+    USD: {code: 'USD', flag: '🇺🇸', name: 'US dollar', symbol: '$', position: 'before'},
+    ZAR: {code: 'ZAR', flag: '🇿🇦', name: 'South African rand', symbol: 'R', position: 'before'},
+}
+
+export function getCurrencyLeftText(currencyCode: string | undefined): string | undefined {
+    if (currencyCode === undefined) {
+        return undefined;
+    }
+    if (currencies[currencyCode] === undefined) {
+        return undefined;
+    }
+    if (currencies[currencyCode].position !== 'before') {
+        return undefined;
+    }
+    return currencies[currencyCode].symbol;
+}
+
+export function getCurrencyRightText(currencyCode: string | undefined): string | undefined{
+    if (currencyCode === undefined) {
+        return undefined;
+    }
+    if (currencies[currencyCode] === undefined) {
+        return undefined;
+    }
+    if (currencies[currencyCode].position !== 'after') {
+        return undefined;
+    }
+    return currencies[currencyCode].symbol;
+}
+
+export function formatCurrencyAmount(currencyCode : string, amount: number): string {
+    const currency = currencies[currencyCode];
+    if (currency.position === 'before') {
```

Comment:
We also need to use optional chaining operator here (`currency?.position` === `before`) operator here otherwise this will throw an error trying to access property of undefined (ofc with branded types, we don't have to care about this ;)